modules/apps/transfer/ibc_module_test.go

```diff
@@ -28,6 +28,11 @@ func (suite *TransferTestSuite) TestOnChanOpenInit() {
 		{
 			"success", func() {}, true,
 		},
+		{
+			"empty version string", func() {
+				channel.Version = ""
+			}, true,
+		},
```

Comment:
Can you also add a test case when `channel.Version` is not empty but it's a string that does not match the expected `types.Version`?

Reply:
We have this test case [already](https://github.com/cosmos/ibc-go/pull/1283/files/0db8c8672e36909b939101d685d19947ffc6f3d9#diff-84e1050545ac3bc948c89e7c6733f8130310338af194e367fa59890ce7796309R52), no?

Reply:
oh, sorry. I missed it.